### DISCUSSION TITLE
Label json files missing image names

### DISCUSSION THREAD

**Discussion by Nadavc220** [bdd100k/bdd100k#171] [Q&A]
Hi,
I am trying to use the dataset with different weathers for semantic segmentation. To my understanding, the weather class of each image is found in its entry in the label files. each entry holds a "name" which holds the name of the file referenced in that entry.

When iterating over the entries if find matches for only 2972 images in the labels file of the training set. for the val set there are no matching entries in the label file, meaning there is no clear mapping between the label file to the actual image names.

Is there something I am missing? is there a way to get the weather class label for all images in the train/val set.

Just to be clear, I tried using the 100k, and 10k image files with the Segmentation labels.

Thanks.

**Comment by iremonur** [comment]
Hi @Nadavc220, I also couldn't find a corresponding image for each segmentation mask. Did you solve the problem?

**Reply by Nadavc220** [reply]
Hi,
I used the data found in [OpenCompoundDomainAdaptation-OCDA](https://github.com/zhmiao/OpenCompoundDomainAdaptation-OCDA).
You can find it under the Data Preparation part, in a link names OCDA Datasets. There you will find the C-Driving benchmark.
